src/v/cluster/tests/id_allocator_stm_test.cc

```diff
@@ -39,20 +39,21 @@ ss::logger idstmlog{"idstm-test"};
 
 struct id_allocator_stm_fixture : simple_raft_fixture {
     void create_stm_and_start_raft() {
-        cfg.id_allocator_batch_size.set_value(int16_t(1));
-        cfg.id_allocator_log_capacity.set_value(int16_t(2));
+        // set configuration parameters
+        config::shard_local_cfg().id_allocator_batch_size.set_value(int16_t(1));
+        config::shard_local_cfg().id_allocator_log_capacity.set_value(
```

Comment:
These should be reset at the end of the test, otherwise other tests in the same process will pick these up. Or consider using a `scoped_config` to set these